src/ucp/tag/rndv.c

```diff
@@ -83,7 +82,7 @@ static size_t ucp_tag_rndv_rts_pack(void *dest, void *arg)
     if (UCP_DT_IS_CONTIG(sreq->send.datatype)) {
         rndv_rts_hdr->address = (uintptr_t) sreq->send.buffer;
         if (ucp_ep_is_rndv_lane_present(ep, 0)) {
-            packed_len += ucp_tag_rndv_pack_rkey(sreq, ucp_ep_get_rndv_get_lane(ep, 0),
+            packed_len += ucp_tag_rndv_pack_rkey(sreq,
```

Comment:
what would be the interface to get length of all keys? Currently `ucp_tag_rndv_pack_rkey` returns md attribute value for one key. Will you add keys packing in the next PR?

Reply:
rkey pack will updated later